### ISSUE TITLE
where is ‘hsi’ in bandmean.m

### ISSUE THREAD

**Issue by caroljaneliu** [qianngli/MCNet#5]
data_pre-processing/band_mean/bandmean.m
you didn't define the parameter hsi in line 19 .Could you tell me what hsi means? Thank you very much:)

**Comment by qianngli** [comment]
Dear reader

we use * load(data_path)  *to load hyperspectral image data.  The data
contains* hsi*. So we do not define it. Of course, you can use other
variables to define it.

OopsCarolLau ***@***.***> 于2021年7月8日周四 下午2:44写道：

**Comment by caroljaneliu** [comment]
i still didn't figure out how the preprocessing stage works. There are three files .  And the last one, band_mean, the hsi which i had asked before didn't exit in training set.   It only has HR and LR.  Or maybe i mistake the loading set in band_mean file. Is it the original training set or the processed training set got from the training_set_augment file?And is your original dataset the .mat file？  Sorry to ask so many questions, im not really get this . Looking forward to your reply!

**Comment by qianngli** [comment]
First, you download the dataset from the corresponding  link. According to
the dataset, you need to divide these datas. In our paper, we randomly
select 80% of the data as the training set and the rest for testing.

With respect to training set, you use *training_set_augment* folder to
augment the dataset by the corresponding  *.m* files. In our work, before
feeding the mini-batch into our network, we subtract the average value of
the entire training images for patches. Therefore,  you need to use *band_mean
*folder to compute the average values. Then, the values of *band_mean* in
the* model.py* need to be replaced. Finally, you can train the model
according to the corresponding dataset.

With respect to test set, you use *test_set_pre-processing* folder to
generate test set according to different scale factors.

Regard,
Li




OopsCarolLau ***@***.***> 于2021年8月13日周五 上午11:31写道：
